### PR TITLE
Including popover to call/whatsapp contact

### LINKED ISSUE #9: [Feature] Call contact (with phone/whatsapp)
**Is your feature request related to a problem? Please describe.**
Currently, the user is not able to call to the contact

**Describe the solution you'd like**
When the user clicks the phone icon, on the contact page, the user should be able to call the contact phone via whatsapp or mobile connection.

**Describe alternatives you've considered**
None.

**Additional context**
Display a message when the call is ended.

component (5)

### PR DESCRIPTION
**Pull request by JoseClaudioADS**
Solves #9

### CODE REVIEW

**Review by fhelipesousa — APPROVED** [review]
Review is ok.

File: src/app/components/call/call.component.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      .callNumber(this.phoneNumber, true)
+      .then(res => {
+      })
+      .catch(err => {
+      });
+    this.popoverRef.dismiss();
+  }
+
+  whatsapp() {
+    window.open(`https://api.whatsapp.com/send?phone=${this.DDI}${this.phoneNumber}`, "_system");
```
Comment (vcasadei) [inline src/app/components/call/call.component.ts]: I believe it's best that you set this URL on a permanent variable of the class.

**Review by vcasadei — CHANGES_REQUESTED** [review]
Tested on Android and iOS and it works.

Just need to fix a minor coding detail.

**Review by vcasadei — APPROVED** [review]
Approved

**Pull request merged**